elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -99,11 +101,13 @@ protected Elide(AuditLogger auditLogger,
             dataStore,
             dictionary,
             mapper,
-            ActivePermissionExecutor::new,
+            permissionExecutor,
             Collections.singletonList(new DefaultFilterDialect(dictionary)),
             Collections.singletonList(new DefaultFilterDialect(dictionary)),
             Pagination.MAX_PAGE_LIMIT,
-            Pagination.DEFAULT_PAGE_LIMIT
+            Pagination.DEFAULT_PAGE_LIMIT,
+            false,
+            HttpStatus.SC_NO_CONTENT
```

Comment:
Should we still have this constructor? I thought Elide constructors were like Highlander…